usage: conda [-h] [--no-plugins] [-V] COMMAND ...
conda: error: argument COMMAND: invalid choice: 'deactivate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'build', 'content-trust', 'convert', 'debug', 'develop', 'doctor', 'index', 'inspect', 'metapackage', 'render', 'skeleton', 'token', 'env', 'server', 'pack', 'verify', 'repo')
usage: conda [-h] [--no-plugins] [-V] COMMAND ...
conda: error: argument COMMAND: invalid choice: 'deactivate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'build', 'content-trust', 'convert', 'debug', 'develop', 'doctor', 'index', 'inspect', 'metapackage', 'render', 'skeleton', 'token', 'env', 'repo', 'server', 'verify', 'pack')
no change     /curc/sw/anaconda3/2023.09/condabin/conda
no change     /curc/sw/anaconda3/2023.09/bin/conda
no change     /curc/sw/anaconda3/2023.09/bin/conda-env
no change     /curc/sw/anaconda3/2023.09/bin/activate
no change     /curc/sw/anaconda3/2023.09/bin/deactivate
no change     /curc/sw/anaconda3/2023.09/etc/profile.d/conda.sh
no change     /curc/sw/anaconda3/2023.09/etc/fish/conf.d/conda.fish
no change     /curc/sw/anaconda3/2023.09/shell/condabin/Conda.psm1
no change     /curc/sw/anaconda3/2023.09/shell/condabin/conda-hook.ps1
no change     /curc/sw/anaconda3/2023.09/lib/python3.11/site-packages/xontrib/conda.xsh
no change     /curc/sw/anaconda3/2023.09/etc/profile.d/conda.csh
no change     /home/mlippincott@xsede.org/.bashrc
No action taken.
[NbConvertApp] WARNING | Config option `kernel_spec_manager_class` not recognized by `NbConvertApp`.
[NbConvertApp] Converting notebook ./notebooks/1.train_regression_multi_output.ipynb to script
[NbConvertApp] Writing 5867 bytes to scripts/1.train_regression_multi_output.py
SHSY5Y False Activin A [NSU]
PBMC False IL-1 beta [NSU]
shuffle: False
Traceback (most recent call last):
  File "/gpfs/alpine1/scratch/mlippincott@xsede.org/Interstellar_Analysis/11.bulk_Morphology_Elastic_Network_LOCO/1.train_models/scripts/1.train_regression_multi_output.py", line 105, in <module>
    data_df = data_df[columns]
NameError: name 'columns' is not defined
Complete
